zephyr/Kconfig

```diff
@@ -46,6 +46,12 @@ if MENDER_MCU_CLIENT
             help
                 Set the Mender server Tenant Token, to be used with https://hosted.mender.io. Retrieve it from the "Organization and billing" settings of your account.
 
+        config MENDER_DEVICE_TYPE
+            string "Mender Device Type"
+            default BOARD_TARGET
```

Comment:
Have you tried this? The idea is not to be literally `BOARD_TARGET` but to use the already defined variable from Zephyr KConfig

Reply:
Yeah, I guess we should do 
```
#ifndef CONFIG_MENDER_DEVICE_TYPE
#define CONFIG_MENDER_DEVICE_TYPE CONFIG_BOARD_TARGET
```

Reply:
Yes, I tried it and it did work just fine. It get's the value from the Zephyr Kconfig

Reply:
Contents in `west build -t menuconfig` after a pristine build:
![Screenshot 2024-10-02 at 16 44 31](https://github.com/user-attachments/assets/670265ec-cb1f-430e-9866-b966f6c062ef)

Reply:
Fantastic. Thank you for double-checking

Reply:
Nice! :heart_eyes: